pages/common/git-stash.md

```diff
@@ -4,24 +4,24 @@
 
 - Stash current changes (except new files):
 
-`git stash save {{optional_stash_name}}`
+`git stash save {{optional_stash_message}}`
 
 - Include new files in the stash (leaves the index completely clean):
 
-`git stash save -u {{optional_stash_name}}`
+`git stash save -u {{optional_stash_message}}`
 
 - List all stashes:
```

Comment:
> Maybe something like "List all stashes with name, related branch and message"?

:+1: 

> Wouldn't that mean that it unstages git-added files as well?

Right. I propose we can leave out "leave the index completely clean". Seems a bit advanced stuff.